scala/BUILD

```diff
@@ -9,6 +9,9 @@ toolchain_type(
 scala_toolchain(
       name = 'default_toolchain_impl',
       scalacopts = [],
+      library = "//external:io_bazel_rules_scala/dependency/scala/scala_library",
```

Comment:
are you sure this is the way to go? If I understand it correctly this still requires me to change the scala.bzl#scala_repositories method to download 2.12.